ktor-server/ktor-server-plugins/ktor-server-compression/jvm/src/io/ktor/server/plugins/compression/Compression.kt

```diff
@@ -12,10 +12,13 @@ import io.ktor.server.request.*
 import io.ktor.server.response.*
 import io.ktor.util.*
 import io.ktor.util.cio.*
+import io.ktor.util.logging.*
 import io.ktor.util.pipeline.*
 import io.ktor.utils.io.*
 import kotlinx.coroutines.*
 
+internal val LOGGER = KtorSimpleLogger("io.ktor.server.plugins.compression.ContentEncoding")
```

Comment:
why it's `ContentEncoding` and not `Compression`?

Reply:
It's a symbol name to lookup. Could you tell me if you think it's better to use Compression here?

Reply:
As far as I see from other PRs on adding Logging, all of them follows the idea that this symbol is the same as name of plugin - which is really useful for understanding on from where it comes from.
F.e. if I would see logger name as `io.ktor.server.plugins.XXXpackage.YYYplugin` and see some issues there, it will be easy to find in which plugin I have an issue.

Reply:
Sure, let me fix this

Reply:
Just in case, what about adding `logger` property just inside plugin builder block?
So it will be super easy for plugin writters to do logs with plugin name out of the box somehow?

Reply:
I'm thinking about this, but it can be confused with the application log in the environment. We also cannot obtain the package name there.

Reply:
Yes, but what if we will provide fully qualified name for plugin name?
Yes it will reduce simplicity, but it can also improve on how attributes work.
I mean now if external plugin will have `Compression` name and is applied in the same scope as `ktor` `Compression` plugin - looks like on of them will replace another

Reply:
I would leave the renaming for a separate task. Not sure if it should be solved along with logging. 
The logger name is fixed

Reply:
yes, of course, sorry, it was just an idea while we are here, not a call to action